app/src/main/java/org/mozilla/focus/browser/Urls.kt

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.browser
+
+/**
+ * A collection of known URLs and parts of URLs.
+ */
+object Urls {
+    const val APP_HOME = "firefox:home"
+    const val APP_PREFIX = "firefox:"
```

Comment:
nit: `private const val SCHEME_APP = "firefox"`

- private: unused elsewhere
- It's more flexible to insert the colon via string interpolation.
- Scheme is more specific